cromulent/model.py

```diff
@@ -223,11 +222,11 @@ def generate_id(self, what):
 
 		return self.base_url + what.__class__._uri_segment + "/" + str(slug)		
 
-	def toJSON(self, what):
+	def toJSON(self, what, done={}):
```

Comment:
I believe (thought this might have changed in python 3.x) that all invocations of `toJSON` *without* a `done` argument here will refer to the same (persistent) dictionary. Safer to have `done=None` and in the function body insatiate the dictionary.